<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QPushButton" name="pb_coffee100">
       <property name="text">
        <string>Coffee(100)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLCDNumber" name="lcdNumber"/>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pb_milk500">
       <property name="text">
        <string>Milk(500)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pb10">
         <property name="text">
          <string>10</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb50">
         <property name="text">
          <string>50</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb100">
         <property name="text">
          <string>100</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb500">
         <property name="text">
          <string>500</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="pb_reset">
       <property name="text">
        <string>reset</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pb_tea300">
       <property name="text">
        <string>Tea(300)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
